cli/src/diff_util.rs

```diff
@@ -1366,9 +1366,14 @@ pub fn show_file_by_file_diff(
                 ui,
                 writer.as_mut(),
                 tool,
+                temp_dir.path(),
                 &maplit::hashmap! {
-                    "left" => left_path.to_str().expect("temp_dir should be valid utf-8"),
-                    "right" => right_path.to_str().expect("temp_dir should be valid utf-8"),
+                    "left" => left_path
+                        .strip_prefix(temp_dir.path()).expect("path should be relative to temp_dir")
+                        .to_str().expect("temp_dir should be valid utf-8"),
```

Comment:
Perhaps this part of the code is a good place to use `Utf8PathBuf` from the `camino` crate, but not in this PR.